.sync.yml

```diff
@@ -43,8 +35,8 @@ spec/spec_helper.rb:
   unmanaged: true
 appveyor.yml:
   matrix_extras:
-    - PUPPET_GEM_VERSION: 3.0.0
-      RUBY_VER: 193
+    - PUPPET_GEM_VERSION: 4.2.3
+      RUBY_VER: 21-x64
```

Comment:
Just out of curiosity, was 4.2.3 chosen at random? Or does it have some significance?